networkit/cpp/distance/HopPlotApproximation.cpp

```diff
@@ -126,9 +126,9 @@ void HopPlotApproximation::run() {
             }
         }
         // add nodes that are already connected to all nodes
-        totalConnectedNodes += (G->numberOfNodes() - activeNodes.size()) * G->numberOfNodes();
+        totalConnectedNodes += static_cast<double>((G->numberOfNodes() - activeNodes.size()) * G->numberOfNodes());
```

Comment:
`G->numberOfNodes()` is used a lot in the code. Better store the result in a `double`-type variable in the beginning and use this. (All other occurences use the result in expressions with double-variables anyway)